.babelrc

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
this is based on our argo uikit babelrc

Reply:
`tsc` doesn't handle CSS insertions, images, etc. which is the reason why also still rely on babel for Arranger Components.
Working on finding a better alternative for ui libraries, but it's in my To Dos list.